packages/sources/coinmetrics-test/src/config/index.ts

```diff
@@ -0,0 +1,63 @@
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+
+export const DEFAULT_API_ENDPOINT = 'https://api.coinmetrics.io/v4'
+export const DEFAULT_WS_API_ENDPOINT = 'wss://api.coinmetrics.io/v4'
```

Comment:
We shouldn't export these so we don't use them by mistake, better to just put the hardcoded string directly in the custom settings